tests/test_setproject.py

```diff
@@ -0,0 +1,45 @@
+from pew._utils import invoke_pew as invoke
+from utils import skip_windows
+
+import tempfile
+import os
+import shutil
+import pytest
+
+
+def test_project_dont_exist(env1):
+    out = invoke('setproject', 'env1', 'i/dont/exist')
+    assert out.err
+
+
+@pytest.yield_fixture()
+def tmp_cwd():
+    tmpdir = tempfile.mkdtemp()
+    old_cwd = os.getcwd()
+    os.chdir(tmpdir)
```

Comment:
I'll probably just use a [TemporaryDirectory](https://docs.python.org/3/library/tempfile.html#tempfile.TemporaryDirectory) here, and create a shim for older pythons in test.util.py